estate/models/estate_property.py

```diff
@@ -0,0 +1,101 @@
+from dateutil.relativedelta import relativedelta
+from odoo import api, models, fields, exceptions
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Property"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    notes = fields.Html()
+    postcode = fields.Char()
+    date_availability = fields.Date(
+        default=fields.Date.today() + relativedelta(months=3), copy=False)
```

Comment:
```suggestion
    date_availability = fields.Date(
        default=lambda self: fields.Date.today() + relativedelta(months=3), copy=False)
```
If you don't use a function here, the date will only be evaluated once when the model is loaded. i.e. if you launch your Odoo on the 01/01 the evaluated date will be 01/03 and will always be that as long as you don't launch it again.